homeassistant/components/intellifire/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Intellifire Config",
+        "description": "Markdown that is shown with the step.",
```

Comment:
I don't think this is the actual string you want in the UI, is it?

Reply:
I removed it.